src/pages/home/sidebar/OptionRow.js

```diff
@@ -93,23 +93,19 @@ const OptionRow = ({
                         )
                     }
                     <View style={[styles.flex1]}>
-                        {(option.text === option.alternateText || option.alternateText.length === 0) ? (
-                            <Text style={[styles.chatSwitcherDisplayName, textUnreadStyle]} numberOfLines={1}>
-                                {option.text}
+                        <Text style={[styles.chatSwitcherDisplayName, textUnreadStyle]} numberOfLines={1}>
+                            {option.text}
+                        </Text>
+                        {option.alternateText ? (
```

Comment:
Does this have the potential to display the `alternateText` twice?

Before, we checked to see if `option.text === option.alternateText` and then just displayed `option.text` if that were the case.

Now that we are always displaying `option.text`, if `option.alternateText` is the same thing we will be displaying it twice, right?

Perhaps this should be `{option.alternateText && option.alternateText !== option.text ? (`.

Reply:
Interesting observation here. I think the `option.alternateText` will never be equal to the `option.text` in the future (this check before was really asking is the `alternateText` and `text` both using the user `login`). So, if it is the same in the future that would mean that somebody entered their own email as a chat message which would be OK.

Reply:
🤷 Okey doke.